.github/workflows/publish_docs.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       uses: conda-incubator/setup-miniconda@v2
       with:
         miniconda-version: "latest"
-        python-version: "3.7"
+        python-version: "3.10"
```

Comment:
Should we immediately go for 3.10. Will this potentially cause trouble while using lower version of python?